applications/nrf_desktop/src/util/CMakeLists.txt

```diff
@@ -4,6 +4,9 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
+target_sources_ifdef(CONFIG_DESKTOP_HID_EVENTQ
+		     app PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/hid_eventq.c)
```

Comment:
cmake indent is 2 spaces, or align with previous line like in c

Reply:
I think it should be already aligned with previous line (github displays it in a strange way)